store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/operation/handler/impl/FederatedAddGraphWithHooksHandlerTest.java

```diff
@@ -76,16 +76,14 @@
     private FederatedStoreProperties federatedStoreProperties;
     private GetAllElements ignore;
 
-    private static Class currentClass = new Object() {
-    }.getClass().getEnclosingClass();
-    private static final AccumuloProperties PROPERTIES = AccumuloProperties.loadStoreProperties(StreamUtil.openStream(currentClass, "properties/singleUseAccumuloStore.properties"));
+    private static final AccumuloProperties PROPERTIES = AccumuloProperties.loadStoreProperties(StreamUtil.openStream(FederatedAddGraphWithHooksHandlerTest.class, "properties/singleUseAccumuloStore.properties"));
```

Comment:
This is inconsistent across all the tests, would be nice to set a precedent and use that

Reply:
The other way using `new Object() { }.getClass().getEnclosingClass();` is ugly and hideous and was way just use copy&paste across classes.

I think using the actual current class is shorter and neater and its how I've done it with FedStore and whenever I use one.

Reply:
Yeah I think this is better, perhaps a tech-def issue should be made for 2.0 to clean this up?